overlord/hookstate/ctlcmd/is_connected.go

```diff
@@ -26,14 +26,18 @@ import (
 	"github.com/snapcore/snapd/interfaces"
 	"github.com/snapcore/snapd/overlord/ifacestate"
 	"github.com/snapcore/snapd/overlord/snapstate"
+	"github.com/snapcore/snapd/sandbox/cgroup"
 )
 
+var cgroupSnapNameFromPid = cgroup.SnapNameFromPid
+
 type isConnectedCommand struct {
 	baseCommand
 
 	Positional struct {
 		PlugOrSlotSpec string `positional-args:"true" positional-arg-name:"<plug|slot>"`
 	} `positional-args:"true" required:"true"`
+	Pid int `long:"pid" description:"Process ID for a connected process"`
```

Comment:
Not sure the description catches the essence. Not happy with my suggestion but what about:
```suggestion
	Pid int `long:"pid" description:"Process ID for a plausibly connected process"`
```